cmd/apichecker/README.md

```diff
@@ -0,0 +1,124 @@
+
+###### Linux/OSX
+GoCryptoTrader is built using [Go Modules](https://github.com/golang/go/wiki/Modules) and requires Go 1.11 or above
+Using Go Modules you now clone this repository **outside** your GOPATH
+
+```bash
+git clone https://github.com/thrasher-corp/gocryptotrader.git
+cd gocryptotrader/cmd/apichecker
+go build
+./apichecker
+```
+
+###### Windows
+
+```bash
+git clone https://github.com/thrasher-corp/gocryptotrader.git
+cd gocryptotrader\cmd\apichecker
+go build && apichecker.exe
+```
+
++ Upon addition of a new exchange, to update Trello checklist and to add the exchange to updates.json the following would need to be done:
+
+###### HTML Scraping method:
+HTMLScrapingData is a struct which contains the necessary information to scrape data from the given path website. Not all the elements of HTMLScrapingData are necessary, its all dependant on site where information is being extracted from. Regexp is used to capture necessary bits of data using r.FindString() where r is the declared regular expression. If update dates data is available, DateFormat is used to convert the dates to a more standard format which can then be used for further comparisons of which update is most recent.
+```go
+func TestAdd(t *testing.T) {
+	t.Parallel()
+	data := HTMLScrapingData{TokenData: "h1",
+		Key:           "id",
+		Val:           "revision-history",
+		TokenDataEnd:  "table",
+		TextTokenData: "td",
+		DateFormat:    "2006/01/02",
+		RegExp:        "^20(\\d){2}/(\\d){2}/(\\d){2}$",
+		CheckString:   "2019/11/15",
+		Path:          "https://docs.gemini.com/rest-api/#revision-history"}
+	err := Add("Gemini", htmlScrape, data.Path, data, true, &testConfigData)
+	if err != nil {
+		t.Error(err)
+    }
+}
+```
+
+###### Github SHA Check Method:
+```go
+func TestAdd(t *testing.T) {
+	t.Parallel()
+	data := GithubData{Repo: "LBank-exchange/lbank-official-api-docs"}
+	err := Add("Lbank", github, fmt.Sprintf(githubPath, data.Repo), data, false, &configData)
+	if err != nil {
+		t.Error(err)
+    }
+}
+```
+
+###### Add using flags:
+```bash
+apichecker.exe -add=true -key=id -val=revision-history -tokendata=h1 -tokendataend=table -texttokendata=td -dateformat=2006/01/02 -checktype="HTML String Check" -regexp="^20(\d){2}/(\d){2}/(\d){2}$" -path="https://docs.gemini.com/rest-api/#revision-history" -exchangename=Gemini
+```
+
++ If all the authentication variables for trello are set trello checklist will be automatically updated with the format of 'Exchange Name (integer of how many updates have been released since the exhange API was last worked on):
+
+- To acquire your trello key and access token please login into trello using the following link and follow the steps: https://trello.com/app-key
+
+- To acquire BoardID, ListID, CardID and ChecklistID the trello api docs can be used: https://developers.trello.com/reference/#api-key-tokens
```

Comment:
I have one last request and that is to make obtaining this super user friendly:

a) The ability for a user to translate a board name to a board ID
b) The ability for a user to get the card list from a board ID: https://developers.trello.com/reference/#boardsboardidtest
c) The ability for a user to get a list ID for a board: https://developers.trello.com/reference/#boardsboardidlists
d) The ability for a user to get a checklist ID from a board ID: https://developers.trello.com/reference/#boardsboardidactions-3

Reply:
e) As discussed, it would be nice to have the ability to create a list, card, checklist for a specified board, plus have it as a test